packages/trader/src/Modules/CFD/Components/cfd-real-account-display.tsx

```diff
@@ -239,7 +299,13 @@ const CFDRealAccountDisplay = ({
             }}
             is_logged_in={is_logged_in}
             existing_data={
-                current_list[Object.keys(current_list).find(key => key.startsWith(`${platform}.real.financial_stp@`))]
+                current_list[
+                    Number(
```

Comment:
Why do we need to convert the string to a number?

Reply:
@balakrishna-binary, I will replace it with string. Typescript doesn’t want to get undefined.